src/components/image-manager/advanced-pull.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+
+import { Button, TextField, withStyles } from '@material-ui/core';
+
+const styles = theme => ({
+  root: {
+    display: 'flex'
+  },
+  imagesListField: {
+    margin: theme.spacing.unit,
+    width: '30ch'
```

Comment:
Do we need to use character width here?

Reply:
Is there a better option to use?

Our default, `openchemistry/chemml:latest`, is 27 characters long. The default is shorter than this, so it cuts off the end. 30 adds a little space at the end and looks nice.